### PR TITLE
Use autoconfig and simplify/clean the prefs

### PR DESCRIPTION
**Pull request by eoger**
Note: I removed some prefs I don't really care about.

### CODE REVIEW

**Comment by eoger** [comment]
Ping!

**Comment by shane-tomlinson** [comment]
@vladikoff - You are probably the best qualified to review this.

**Comment by vladikoff** [comment]
I will take a look at this soon, thanks

File: profile.js [hunk 1]
```diff
@@ -2,68 +2,19 @@ var path = require('path');
...
-  }
+  'local': 'http://127.0.0.1:3030',
+  'latest': 'https://latest.dev.lcip.org',
+  'stable': 'https://stable.dev.lcip.org',
+  'stage': 'https://accounts.stage.mozaws.net',
+  'prod': 'https://accounts.firefox.com'
 };
 
 var env = process.env.FXA_ENV || 'local';
-var FXA_DESKTOP_CONTEXT = process.env.FXA_DESKTOP_CONTEXT || 'fx_desktop_v3';
```
Comment (vladikoff) [inline profile.js:51]: I think we still need a way to be able to control this `FXA_DESKTOP_CONTEXT`

Comment (vladikoff) [reply profile.js:51]: @shane-tomlinson thoughts?

**Comment by eoger** [comment]
Besides `FXA_DESKTOP_CONTEXT` is there anything else that needs to be addressed to move this PR forward?

**Comment by vladikoff** [comment]
I will take a look !

**Pull request merged**